checkov/terraform/checks/resource/gcp/GoogleCloudMySqlLocalInfileOff.py

```diff
@@ -0,0 +1,38 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class GoogleCloudMySqlLocalInfileOff(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure MySQL database 'local_infile' flag is set to 'off'"
+        check_id = "CKV_GCP_50"
+        supported_resources = ['google_sql_database_instance']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=check_id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        """
+            Looks for google_sql_database_instance which enables local_infile:
+            :param
+            conf: google_sql_database_instance
+            configuration
+            :return: < CheckResult >
+        """
+        if 'database_version' in conf.keys():
+            key = conf['database_version'][0]
+            if key=='MYSQL_5_6' or key=='MYSQL_5_7' or key=='MYSQL_8_0':
```

Comment:
In all of the examples like the three lines above, it'd be better to create a few lists of approved / restricted values for the different sql dbs, and then just compare on value to the imported list. It would be easier to read, and whenever the list should be updated we can update it only in one place.

e.g:
mysql_version = ['MYSQL_5_6', 'MYSQL_5_7', 'MYSQL_8_0']
**
import mysql_version
**
if 'database_version' in conf.keys() and conf['database_version'][0] in mysql_versions:

Reply:
Better, why not just check if starts with MYSQL, without the version? the list is updating, so we don't even need to track it